clang/test/dpct/cmake_migration/case_038/expected.txt

```diff
@@ -0,0 +1,3 @@
+target_link_directories(CUDA::toolkit INTERFACE "${ONEAPI_LIB_DIR}")
+
+target_link_options(quda PUBLIC )
```

Comment:
is the expected.txt runnable with CMake?

Reply:
Yes